phovea_server/dataset_csv.py

```diff
@@ -58,9 +59,9 @@ def load(self):
       return self._loaded
 
     data = []
-    with open(self._path, 'r') as csvfile:
-      reader = csv.reader(csvfile, delimiter=self._desc.get('separator', ',').encode('ascii', 'ignore'),
-                          quotechar=str(self._desc.get('quotechar', '|')).encode('ascii', 'ignore'))
+    with io.open(self._path, 'r', newline='', encoding=self._desc.get('encoding', 'utf-8')) as csvfile:
```

Comment:
why is setting the newline to '' important?

Reply:
Yes, according to the [backport.csv](https://pypi.python.org/pypi/backports.csv):

> Note: It should always be safe to specify newline='', since the csv module does its own (universal) newline handling.